applications/acdc/src/acdc_queue_listener.erl

```diff
@@ -353,6 +357,14 @@ handle_cast({'timeout_member_call'}, #state{delivery=Delivery
     acdc_util:unbind_from_call_events(Call),
     lager:debug("unbound from call events for ~s", [whapps_call:call_id(Call)]),
 
+    case AgentId of
```

Comment:
Put this in a function like `maybe_timeout_agent(AgentId, Call, JObj)` and turn the case clauses into function clauses.